packages/mobile/src/styles/palettes/default.palette.ts

```diff
@@ -34,5 +34,8 @@ export const defaultPalette = {
     gray70: '#4C4C4C',
     blue: '#2373EA',
     lushSky: '#67BFD3'
+  },
+  pendingMessage: {
```

Comment:
I guess we should rather add it as one of typography colors. The hierarchy of this palette is different 'components' has its own class (eg typography, input, background) and colors defined within those classes